src/api/python/cvc5.pxd

```diff
@@ -388,6 +389,8 @@ cdef extern from "api/cpp/cvc5.h" namespace "cvc5::api":
             Term operator*() except +
         const_iterator begin() except +
         const_iterator end() except +
+
+        bint isConstArray() except +
```

Comment:
What about `getConstArrayBase()`?

Reply:
It was already implemented some place else in the file so wasn't visible in the diff. But now I moved it here.